eml/api/api.js

```diff
@@ -2,7 +2,8 @@ import axios from 'axios';
 
 const timeoutInMs = 1200;
 
-const url = 'http://172.30.245.212:8888'; // change to lcd ip when testing
+const url = 'https://educado-backend-staging-x7rgvjso4a-ew.a.run.app/';
```

Comment:
Is it possible to make a .env file for the URLs? And why the staging URL when we're not trying to merge to staging?

Reply:
i think it is an integration issue and not related to this PR. this is meant to go to staging at some point, thats why its the staging url and not local